atlasdb-commons/src/main/java/com/palantir/async/initializer/AsyncInitializer.java

```diff
@@ -77,40 +70,35 @@ default void scheduleInitialization() {
                     scheduleInitialization();
                 }
             }
-        }, millisUntilNextAttempt, TimeUnit.MILLISECONDS);
+        }, sleepIntervalInMillis(), TimeUnit.MILLISECONDS);
     }
 
-    default int millisToNextAttempt() {
-        return millisUntilNextAttempt;
+    @VisibleForTesting
+    protected int sleepIntervalInMillis() {
+        return 10_000;
     }
 
-    // TODO (JAVA9): Make this private.
-    default void tryToInitializeIfNotInitialized() {
+    private void tryToInitializeIfNotInitialized() {
         if (isInitialized()) {
             log.warn("{} was initialized underneath us.",
```

Comment:
does this need to be a warn? if we expect this to be able to happen (i.e., if something synchronously inits something while there is an async init task scheduled), i think we should just be silent here or log something that is not scary